app/README.md

```diff
@@ -34,10 +36,10 @@ For production builds, vite preforms a string replacement on the `import.meta.en
 
 The table below describes the environment variables that are used by this app:
 
-| Variable | Required? | Example | Description |
-|----------|-----------|---------|-------------|
-| `VITE_HUU_API_BASE_URL` | YES | http://localhost:4040/api/ | The HUU API's base URL. In a development environment (mode is 'development' or 'test'): if this variable is not defined, then `http://localhost:4040/api/` will be used by default. In non-development environment: if this variable is not defined, then the build will throw an error. |
-|          |           |         |             |
+| Variable                | Required? | Example                    | Description                                                                                                                                                                                                                                                                              |
+| ----------------------- | --------- | -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `VITE_HUU_API_BASE_URL` | YES       | http://localhost:4040/api/ | The HUU API's base URL. In a development environment (mode is 'development' or 'test'): if this variable is not defined, then `http://localhost:4040/api/` will be used by default. In non-development environment: if this variable is not defined, then the build will throw an error. |
+|                         |           |                            |                                                                                                                                                                                                                                                                                          |
```

Comment:
Optional, but can you revert this change? The extra spaces and `-` are ignored by the markdown renderer and there is an extra empty row now.

Reply:
Yes! I don't even know what happened here, good catch.